etc/config-template.toml

```diff
@@ -99,10 +99,7 @@
 # region-split-size = "96MB"
 # When region size changes exceeds region-split-check-diff, we should check
 # whether the region should be split or not.
-# region-split-check-diff = "12MB"
-# Set it to true will check regions if they need to spilt right after
-# initialization. It is useful when we adjust the region size.
-# region-split-check-after-initialization = false
+# region-split-check-diff = "6MB"
```

Comment:
I guess 6MB and 12MB have no big difference.

Reply:
Maybe it is better to still use 12MB here.

Reply:
A smaller diff makes the cached region size more accurate. Since the split check will always check the approximate region size before scanning the region, a smaller diff will not lead to much overhead.